### ISSUE TITLE
BusyIndicator changed Content's Background to White when IsBusy is TRUE 

### ISSUE THREAD

**Issue by kuchienkz** [xceedsoftware/wpftoolkit#1331]
I have a ListView with Background set to green, then I put it inside BusyIndicator control. When IsBusy is set to TRUE, the ListView's Background becomes white, and go back to green when IsTrue is set to FALSE.

Is this normal? How can I keep the ListView's Background while IsBusy is set to TRUE ??

**Comment by XceedBoucherS** [comment]
Hi,

This doesn't happens in the following sample:
` <StackPanel Margin="10">
    <CheckBox x:Name="_checkbox"
              IsChecked="False" />
    <TextBlock Text="Custom Content:" />
    <xctk:BusyIndicator IsBusy="{Binding IsChecked, ElementName=_checkbox}">
      <xctk:BusyIndicator.BusyContent>
        <ListView Background="Green"
                  Height="100"
                  Width="100" />
      </xctk:BusyIndicator.BusyContent>
    </xctk:BusyIndicator>
  </StackPanel>`

**Comment by kuchienkz** [comment]
Yes that will work, i've tried it before, im just having the problem with a certain ListView (with custom panel and item template).

**Comment by XceedBoucherS** [comment]
Could you paste a sample with this issue ?

**Comment by kuchienkz** [comment]
Im using [this](https://github.com/kuchienkz/VirtualizingWrapPanel/blob/master/VirtualizingWrapPanel.cs) panel, please try it.

**Comment by XceedBoucherS** [comment]
Everything looks normal in v3.3 with the following code : 
` <StackPanel Margin="10">
    <CheckBox x:Name="_checkbox"
              IsChecked="False" />
    <TextBlock Text="Custom Content:" />
    <xctk:BusyIndicator IsBusy="{Binding IsChecked, ElementName=_checkbox}">
      <xctk:BusyIndicator.BusyContent>
        <ListView Background="Green"
                  Height="100"
                  Width="100"
                  ItemsSource="{Binding MyData}">
          <ListView.ItemsPanel>
            <ItemsPanelTemplate>
              <local:VirtualizingWrapPanel />
            </ItemsPanelTemplate>
          </ListView.ItemsPanel>
        </ListView>
      </xctk:BusyIndicator.BusyContent>
    </xctk:BusyIndicator>
  </StackPanel>`
